File: lib/src/models/event.dart
class Event {
  final String title;
  final String description;
  DateTime date;
  DateTime time;
  final String owner;

  Event({
    required this.title,
    required this.description,
    required this.date,
    required this.time,
    required this.owner,
  });
  @override
  String toString() => title;
}

File: APIs/TutorMe/TutorMe/Models/Event.cs
using System.Text.Json.Serialization;

namespace TutorMe.Models {
    public class Event {
        public Guid EventId { get; set; }
        public Guid? GroupId { get; set; }
        public Guid OwnerId { get; set; }
        public Guid UserId { get; set; }
        public string DateOfEvent { get; set; }
        public string VideoLink { get; set; }
        public string TimeOfEvent { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }



        [JsonIgnore]
        public virtual Group Group { get; set; }
        [JsonIgnore]
        public virtual User User { get; set; }
        [JsonIgnore]
        public virtual User Owner { get; set; }
    }
}
